functions/__tests__/functions.test.ts

```diff
@@ -0,0 +1,134 @@
+import * as functionsTestInit from 'firebase-functions-test'
+import mockedEnv from 'mocked-env'
+import { mocked } from 'ts-jest/utils'
+import { MeiliSearch } from 'meilisearch'
+
+jest.mock('meilisearch')
+
+const defaultEnvironment = {
+  LOCATION: 'us-central1',
+  PROJECT_ID: 'fake-project',
+  COLLECTION_PATH: 'example',
+  INPUT_FIELD_NAME: '',
+  MEILISEARCH_INDEX_NAME: 'example',
+  MEILISEARCH_HOST: 'http://127.0.0.1:7700',
+  MEILISEARCH_API_KEY: 'masterKey',
+}
+
+export const mockExport = (document, data) => {
+  const ref = require('../src/index').indexingWorker
+  const functionsTest = functionsTestInit()
+
+  const wrapped = functionsTest.wrap(ref)
+  return wrapped(document, data)
+}
+
+let restoreEnv
```

Comment:
This only used in 
```
  beforeEach(() => {
    restoreEnv = mockedEnv(defaultEnvironment)
  })
```
I'm not sure I understand the purpose of it
```suggestion
```

Reply:
`It modifies process.env per passed arguments, calls a designated callback, then restores process.env to its original value.`
https://github.com/burl/mock-env
I think it should be instantiated outside of the test to be called each time in `beforeEach`

Reply:
I removed it and the tests did not fail does `process.env` needs to be restored?

Reply:
Ah my bad I tried it again and indeed it fails! Yes putting it inside the describe is better so that all the variables are together

Reply:
We let it like that for clarity and to maybe use:
`afterEach(() => restoreEnv())`

Reply:
My bad I edit my message with the good link to the documentation on `mocked-env`

Reply:
After discussion it should be removed and only added in `beforeEach` without a variable